skbase/base/_base.py

```diff
@@ -260,6 +257,11 @@ def _clone(self, estimator, *, safe=True):
                     "Cannot clone object %s, as the constructor "
                     "either does not set or modifies parameter %s" % (estimator, name)
                 )
+
+        # This is an extension to the original sklearn implementation
+        if isinstance(estimator, BaseObject) and estimator.get_config()["clone_config"]:
```

Comment:
I assume this is now WiP, just wanted to mention that anything you move after the lines
```python
        if not self.get_config()["check_clone"]:
            return self_clone
```
will not happen if `check_clone` is off, and it is off by default.

Reply:
I think that you misread the change. This change is inside `_clone()` which is called **before** that _if_-block. Does that make sense?

Reply:
oh, yes, I must have somehow hallucinated the change somewher else. You are right.